node/src/server-modules/GlideJson.ts

```diff
@@ -0,0 +1,164 @@
+/**
```

Comment:
@shohamazon we used the module name "GlideJson" so that we don't conflict with standard json libraries.  We should consider doing the same for Python so as not to conflict with the python json libraries that will probably be imported at the same time.

Reply:
yeah no worry